packages/clerk-js/src/ui/common/RemoveResourcePage.tsx

```diff
@@ -39,14 +44,10 @@ export const RemoveResourcePage = withCardStateProvider((props: RemovePageProps)
           <Text localizationKey={messageLine2} />
           <FormButtons
             variant='primaryDanger'
-            onReset={close}
+            onReset={onReset || close}
           />
         </Form.Root>
       </FormContent>
-      <SuccessPage
-        title={title}
-        text={successMessage}
-      />
```

Comment:
Are we removing this?

Reply:
No designs for it. I'll remove those from the codebase in another PR.